API/src/Modules/Reports/Momentum.Reports.Infrastructure/MartenInitializer.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Marten;
+using Momentum.Framework.Core.DependencyInjection;
+using Momentum.Reports.Core.Models;
+
+namespace Momentum.Reports.Infrastructure
+{
+    public class MartenInitializer : IMartenInitializer
+    {
+        public void SetupDomainIdentities(StoreOptions options)
+        {
+            options.Schema.For<Report>().Identity(x => x.Id);
```

Comment:
In this case you don't need the line as Marten automatically matches the `Id` property, see [here](https://martendb.io/documentation/documents/identity/)